styles.css

```diff
@@ -0,0 +1,144 @@
+:root {
+  --bright-orange: hsl(31, 77%, 52%);
+  --dark-cyan: hsl(184, 100%, 22%);
+  --very-dark-cyan: hsl(179, 100%, 13%);
+  --neutral-white: hsla(0, 0%, 100%, 0.75);
+  --light-grey: hsl(0, 0%, 95%);
+  --transparent-white: hsla(0, 0%, 100%, 0.75);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lexend Deca", sans-serif;
+}
+
+picture {
+  padding-bottom: 50px;
+  margin-bottom: 50px;
+}
+
+h1 {
+  font-family: "Big Shoulders Display", cursive;
+  text-transform: uppercase;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  color: var(--light-grey);
+}
+
+p {
+  margin-bottom: 30px;
+  color: var(--transparent-white);
+}
+
+button {
+  padding: 15px;
+  border-radius: 30px;
+  border-style: solid;
+  font-family: "Lexend Deca", sans-serif;
+}
+
+.columns {
+  grid-template-rows: 3 (1fr);
+  grid-template-columns: 1fr;
+  display: grid;
+  /* align-items: center;
+    justify-content: center; */
```

Comment:
It's a good idea to remove unused code as it improves readability and maintenance.
```suggestion
```